api/include/opentelemetry/logs/severity.h

```diff
@@ -16,8 +16,8 @@ namespace logs
  */
 enum class Severity : uint8_t
 {
-  kInvalid,
-  kTrace,
+  kInvalid = 0,
+  kTrace   = 1,
```

Comment:
I am fine on each way. For the current change, `kInvlid` is assigned 0 because it is not given in the OTel spec, and only created for us, so makes it explicit. for `kTrace`, it is the first severity defined in OTel spec with value. It doesn't break the spec if we change the value of `kInvalid` later.

Reply:
+1 on being explicit

Reply:
Assigned explicit number value to each severity level.